hadoop-ozone/dist/src/main/compose/ozonesecure-ha/test-scm-decommission.sh

```diff
@@ -47,4 +47,4 @@ SCMID=$(execute_command_in_container scm4.org bash -c "ozone admin scm roles" |
 docker-compose stop scm4.org
 execute_robot_test scm3.org kinit.robot
 wait_for_execute_command scm3.org 60 "ozone admin scm decommission --nodeid=${SCMID} | grep Decommissioned"
```

Comment:
I'm guessing `kinit` is required for the commands in `scm-decommission.robot`, too.  Either all 3 steps should be executed on `s3g`, or better yet, `Kinit` test setup step should be added to `scm-decommission.robot` like this:

https://github.com/apache/ozone/blob/2af0e64675aaad616cff1952eb0b6c5c12ceea21/hadoop-ozone/dist/src/main/smoketest/omha/om-fetch-key.robot#L21

Reply:
Sorry, my bad, missed this one.
Updating the PR

Reply:
Updated the PR. Tested locally on s3g, works fine.

```
export SECURITY_ENABLED=true
❯ ../test-single.sh s3g scmha/scm-decommission.robot
==============================================================================
Scm-Decommission :: Test Ozone SCM Decommissioning                            
==============================================================================
Decommission SCM Primordial Node                                      | PASS |
------------------------------------------------------------------------------
Scm-Decommission :: Test Ozone SCM Decommissioning                    | PASS |
1 test, 1 passed, 0 failed
==============================================================================
```

Reply:
Started new CI runs on my fork, cancelled old ones. Thanks!

Reply:
CI Passed: https://github.com/LZD-PratyushBhatt/ozone/actions/runs/6505325429/job/17669363810?pr=8

Reply:
@adoroszlai can you please review again? Thanks!